src/shared/routes.ts

```diff
@@ -121,6 +121,7 @@ export const routes: IRoutePropsWithFetch<RouteData, any, any>[] = [
     component: CreatePost,
     fetchInitialData: CreatePost.fetchInitialData,
     getQueryParams: getCreatePostQueryParams,
+    mountedSameRouteNavKey: "create_post",
```

Comment:
What does this do?

Reply:
In `App` routes without that property get the `location.key` assigned as key, so that they unmount whenever the location changes. Before that property all routes behaved like that.